eng/pipelines/templates/scripts/validate-changes-on-allowed-path.ps1

```diff
@@ -0,0 +1,64 @@
+<#
+.SYNOPSIS
+
+.DESCRIPTION
+The script is to validate if the diff changes between base and target are within the scope of allowlist.
+
+.PARAMETER BaseBranch
+The base branch where we want to merge changes from. E.g. main branch in docs repo
+
+.PARAMETER TargetBranch
+The target branch where we want to merge changes to. E.g. live branch in docs repo
+
+.PARAMETER AllowlistPath
+The file path of allowlist.
+#>
+param(
+  [Parameter(mandatory=$true)]
+  [string] $BaseBranch,
+  [Parameter(mandatory=$true)]
+  [string] $TargetBranch,
+  [Parameter(mandatory=$true)]
+  [string] $AllowlistPath
+)
+
+# Fetch all branches, e.g. main, live
+Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" fetch origin"
+git -c user.name="azure-sdk" -c user.email="azuresdk@microsoft.com" fetch origin | Write-Host
```

Comment:
I'm not sure I understand the pipe to write-host pattern I see here is there a reason for doing that?

Reply:
I personally put `| Write-Host` to the places easy to fail. The detail info about git doesn't introduce too many overheads to my pipeline, but really helpful to learn the behavior at the first setup.

Reply:
I don't expect this to be necessary to get the output on the console and in the logs, it should already show up.

Reply:
I have the piece of codes originally in window agent inline powershell. They swallowed my git message somehow. I will test in our 1ES agent pool to see if they can have git messages directly without write-host